getting_started/step_by_step/ui_main_menu.rst

```diff
@@ -204,7 +204,7 @@ need two containers as children of our ``HBoxContainer``: a
    sitting aside from it
 
 In the node tree, select all the ``TextureRect`` nodes that should go on
-the left side: the logo, the menu options and the version note. Drag and
+the left side: the logo, the menu options (Continue, NewGame, Options) and the version note. Drag and
```

Comment:
This should also have an Oxford comma (as that's the standard we've adopted for the docs).

Reply:
What Chris means is we should have a comma before "and", like so:

> menu options (Continue, NewGame, Options), and the